spec/requests/user_request_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 RSpec.describe "art requests", type: :request do
 
-  before(:all) do
+  before(:each) do
```

Comment:
i noticed my extra tests were utilizing the same user id as the first, existing test. no bueno!

Reply:
I noticed the `before(:all`) in other places as well and was surprised. This seems like a very good change!